test/e2e/federation-configmap.go

```diff
@@ -0,0 +1,269 @@
+/*
```

Comment:
We decided to name these test/e2e/federated-* instead of federation-*. We havent renamed all existing files yet, but we want to follow that pattern for new files